app/assets/javascripts/components/widget-report.js

```diff
@@ -0,0 +1,39 @@
+ManageIQ.angular.app.component('widgetReport', {
+  bindings: {
+    id: '<',
+  },
+  controllerAs: 'vm',
+  controller: ['$http', 'miqService', '$sce', function($http, miqService, $sce) {
+    var vm = this;
+    vm.widgetReportModel = {};
+
+    this.$onInit = function() {
+      $http.get('/dashboard/widget_report_data/' + vm.id)
+        .then(function(response) { vm.widgetReportModel.content = $sce.getTrustedHtml(response.data.content);})
```

Comment:
Agreed, `getTrustedHtml` is definitely safer, it doesn't seem to remove colors from reports.. let's keep it until someone complains :)